ui/imports/shared/views/AssetsView.qml

```diff
@@ -62,42 +69,78 @@ ColumnLayout {
             settingsKey: "WalletAssets"
         }
 
-        readonly property bool hasCommunityAssets: communityAssetsLV.count
-
         function hideAllCommunityTokens(communityId) {
-            const tokenSymbols = ModelUtils.getAll(communityAssetsLV.model, "symbol", "communityId", communityId)
+            const tokenSymbols = ModelUtils.getAll(regularAssetsLV.model, "symbol", "communityId", communityId)
```

Comment:
Why the change here?

Reply:
https://github.com/status-im/status-desktop/pull/12871#discussion_r1434200352